src/pybind/mgr/stats/fs/perf_stats.py

```diff
@@ -124,6 +124,13 @@ def set_client_metadata(self, client_id, key, meta):
             result[key] = meta
 
     def notify(self, cmdtag):
+        try:
+            uuid.UUID(str(cmdtag))
```

Comment:
You need to check if `cmdtag` is a valid key in `self.client_metadata['in_progress']`.